assignment1.txt

```diff
@@ -0,0 +1,8 @@
+Albert Zhang
+1. A compiler is what turns the code that is readable by humans into something that the computer can easily read and turn into binary.
```

Comment:
sort of, more specifically, a compiler takes your code written in C, removes the whitespace, places this into a parse tree, optimizes it, and then takes that parse tree and translates it into assembly code.